astroquery/image_cutouts/first/core.py

```diff
@@ -49,7 +49,7 @@ def _args_to_payload(self, coordinates, image_size=1 * u.arcmin,
         return request_payload
 
     @prepend_docstr_nosections("\n" + _args_to_payload.__doc__)
-    def get_images(self, coordinates, image_size=1 * u.arcmin,
+    def get_images(self, coordinates, *, image_size=1 * u.arcmin,
```

Comment:
I approve of the change, *but* would it make sense, in this case, to have the `*` after `image_size`, since image size is reasonably paired with `coordinates`?  Arguably, we shouldn't even have a default for image size...

Reply:
Sure, that works for me, too. Whether a default makes sense or not I'm fine with either way, but let me know whether to change it in this PR.

Reply:
let's keep the default, but also keep the order of arguments: that allows `image_size` to be specified as either a kwarg or positional, which I think is reasonable enough here.